### PR TITLE
v1.19.0 - Logo size tweaks

### PR DESCRIPTION
**Pull request by xander-marjoram**
Fixing the logo sizing now that I have the designs.
Recommend hiding whitespace changes when reviewing the CSS file.

## Browsers Tested
- [x] Chrome
- [ ] Edge
- [ ] Internet Explorer 11
- [ ] Mobile (i.e. iPhone/Android - please list device)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: src/scss/partials/_logo.scss [hunk 1]
```diff
@@ -31,39 +31,29 @@ $logo-color--transparent    : $white;
...
-        @if ($theme == 'ml') {
-          path:first-child {
-            .c-header--transparent & {
-              display: none;
-            }
-          }
+        @include media('>=mid') {
+          width: 164px;
+          height: 46px;
+          margin-left: 0;
```
Comment (siggerzz) [inline src/scss/partials/_logo.scss:55]: I know this code was there previously, but just a question as my CSS knowledge is limited.

Why do we specify `margin-left` if we're not adding any margin? Is it purely there to stop it getting inherited from somewhere else?

Comment (xander-marjoram) [reply src/scss/partials/_logo.scss:55]: @siggerzz Yep that's right, so if you look above there is a `margin-left: -10.5px;` by default for Menulog, but here for screen widths larger than the mid breakpoint, we basically remove that rule by setting it back to zero

File: src/scss/partials/_logo.scss [hunk 2]
```diff
@@ -31,39 +31,29 @@ $logo-color--transparent    : $white;
...
-            height: 26px;
-            margin-left: -10.5px; //half of hamburger menu width
+    @include media('>=mid') {
+        width: 165px;
+        height: 32px;
+    }
 
+      // Menulog logo, as it has multiple fill values built in. We just hide the outline on transparent mode.
+    @if ($theme == 'ml') {
+        width: 92px;
```
Comment (ashleynolan) [inline src/scss/partials/_logo.scss:47]: Should this not be `120 x 32` on mobile, and then below `149 x 41`?

Comment (xander-marjoram) [reply src/scss/partials/_logo.scss:47]: I was going by what's on Zeplin, please could you show me where those dimensions are from?

**Pull request merged**